2022-03-26 21:50:30,274 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/.config/wandb/settings
2022-03-26 21:50:30,275 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/settings
2022-03-26 21:50:30,275 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-26 21:50:30,275 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-26 21:50:30,275 INFO    MainThread:12299 [wandb_init.py:_log_setup():405] Logging user logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220326_215030-3037kpd7/logs/debug.log
2022-03-26 21:50:30,275 INFO    MainThread:12299 [wandb_init.py:_log_setup():406] Logging internal logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220326_215030-3037kpd7/logs/debug-internal.log
2022-03-26 21:50:30,275 INFO    MainThread:12299 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f83b722d100>
2022-03-26 21:50:30,275 INFO    MainThread:12299 [wandb_init.py:init():439] calling init triggers
2022-03-26 21:50:30,275 INFO    MainThread:12299 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-26 21:50:30,275 INFO    MainThread:12299 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 234t2r2c
2022-03-26 21:50:30,277 INFO    MainThread:12299 [wandb_run.py:finish():1539] finishing run afischer/tcn_samp_en-project/234t2r2c
2022-03-26 21:50:30,277 INFO    MainThread:12299 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-26 21:50:30,279 INFO    MainThread:12299 [wandb_run.py:_restore():1674] restore
2022-03-26 21:50:30,667 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-26 21:50:30,891 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-26 21:50:31,134 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 113227
}

2022-03-26 21:50:31,243 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 119123
}

2022-03-26 21:50:31,349 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 119123
}

2022-03-26 21:50:31,456 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 119123
  total_bytes: 119123
}

2022-03-26 21:50:31,566 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 119123
  total_bytes: 119123
}

2022-03-26 21:50:31,671 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 119123
  total_bytes: 119123
}

2022-03-26 21:50:31,778 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 119123
  total_bytes: 119123
}

2022-03-26 21:50:31,886 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 119123
  total_bytes: 119123
}

2022-03-26 21:50:31,990 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 119123
  total_bytes: 119123
}

2022-03-26 21:50:32,098 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 119123
  total_bytes: 119123
}

2022-03-26 21:50:32,950 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 119123
  total_bytes: 119123
}

2022-03-26 21:50:33,428 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 119123
  total_bytes: 119123
}
local_info {
}

2022-03-26 21:50:34,568 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-26 21:50:34,570 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-26 21:50:34,573 INFO    MainThread:12299 [wandb_run.py:_footer_sync_info():2822] logging synced files
2022-03-26 21:50:34,575 INFO    MainThread:12299 [wandb_init.py:init():492] starting backend
2022-03-26 21:50:34,575 INFO    MainThread:12299 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-03-26 21:50:34,577 INFO    MainThread:12299 [backend.py:ensure_launched():219] starting backend process...
2022-03-26 21:50:34,595 INFO    MainThread:12299 [backend.py:ensure_launched():224] started backend process with pid: 13326
2022-03-26 21:50:34,596 INFO    MainThread:12299 [wandb_init.py:init():501] backend started and connected
2022-03-26 21:50:34,616 INFO    MainThread:12299 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-03-26 21:50:34,617 INFO    MainThread:12299 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-26 21:50:34,617 INFO    MainThread:12299 [wandb_init.py:init():565] updated telemetry
2022-03-26 21:50:34,651 INFO    MainThread:12299 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-26 21:50:35,490 INFO    MainThread:12299 [wandb_run.py:_on_init():1759] communicating current version
2022-03-26 21:50:35,534 INFO    MainThread:12299 [wandb_run.py:_on_init():1763] got version response 
2022-03-26 21:50:35,534 INFO    MainThread:12299 [wandb_init.py:init():625] starting run threads in backend
2022-03-26 21:50:37,672 INFO    MainThread:12299 [wandb_run.py:_console_start():1733] atexit reg
2022-03-26 21:50:37,673 INFO    MainThread:12299 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-03-26 21:50:37,673 INFO    MainThread:12299 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-03-26 21:50:37,674 INFO    MainThread:12299 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-26 21:50:37,674 INFO    MainThread:12299 [wandb_init.py:init():664] run started, returning control to user process
2022-03-26 21:50:37,674 INFO    MainThread:12299 [wandb_watch.py:watch():47] Watching
2022-03-26 21:52:56,694 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set batch_size = 60 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83a27987f0>>
2022-03-26 21:52:56,694 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb batch_size 60 None
2022-03-26 21:52:56,695 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set epochs = 45 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83a27987f0>>
2022-03-26 21:52:56,695 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb epochs 45 None
2022-03-26 21:52:56,695 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set lr = 0.0014669818184550845 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83a27987f0>>
2022-03-26 21:52:56,695 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb lr 0.0014669818184550845 None
2022-03-26 21:52:56,695 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set no_cuda = False - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83a27987f0>>
2022-03-26 21:52:56,695 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb no_cuda False None
2022-03-26 21:52:56,695 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set seed = 42 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83a27987f0>>
2022-03-26 21:52:56,695 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb seed 42 None
2022-03-26 21:53:28,236 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/.config/wandb/settings
2022-03-26 21:53:28,237 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/settings
2022-03-26 21:53:28,237 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-26 21:53:28,237 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-26 21:53:28,237 INFO    MainThread:12299 [wandb_init.py:_log_setup():405] Logging user logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220326_215328-1fon5vec/logs/debug.log
2022-03-26 21:53:28,237 INFO    MainThread:12299 [wandb_init.py:_log_setup():406] Logging internal logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220326_215328-1fon5vec/logs/debug-internal.log
2022-03-26 21:53:28,237 INFO    MainThread:12299 [wandb_init.py:init():439] calling init triggers
2022-03-26 21:53:28,237 INFO    MainThread:12299 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-26 21:53:28,237 INFO    MainThread:12299 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 3037kpd7
2022-03-26 21:53:28,239 INFO    MainThread:12299 [wandb_run.py:finish():1539] finishing run afischer/tcn_samp_en-project/3037kpd7
2022-03-26 21:53:28,240 INFO    MainThread:12299 [jupyter.py:save_history():429] not saving jupyter history
2022-03-26 21:53:28,240 INFO    MainThread:12299 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-26 21:53:28,240 INFO    MainThread:12299 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-03-26 21:53:28,240 INFO    MainThread:12299 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-26 21:53:28,243 INFO    MainThread:12299 [wandb_run.py:_restore():1674] restore
2022-03-26 21:53:28,482 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-26 21:53:28,617 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-26 21:53:29,410 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-26 21:53:30,327 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 126069
}

2022-03-26 21:53:30,434 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 126069
}

2022-03-26 21:53:30,538 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 126069
}

2022-03-26 21:53:30,645 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 126069
  total_bytes: 126069
}

2022-03-26 21:53:30,749 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 126069
  total_bytes: 126069
}

2022-03-26 21:53:30,856 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 126069
  total_bytes: 126069
}

2022-03-26 21:53:30,961 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 126069
  total_bytes: 126069
}

2022-03-26 21:53:31,065 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 126069
  total_bytes: 126069
}

2022-03-26 21:53:31,182 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 126069
  total_bytes: 126069
}

2022-03-26 21:53:31,297 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 126069
  total_bytes: 126069
}

2022-03-26 21:53:31,643 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 126069
  total_bytes: 126069
}

2022-03-26 21:53:32,112 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 126069
  total_bytes: 126069
}
local_info {
}

2022-03-26 21:53:33,299 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-26 21:53:33,301 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-26 21:53:33,304 INFO    MainThread:12299 [wandb_run.py:_footer_sync_info():2822] logging synced files
